ci-operator/step-registry/cucushift/upgrade/rhel-worker/cucushift-upgrade-rhel-worker-commands.sh

```diff
@@ -330,13 +368,17 @@ export SOURCE_VERSION
 export SOURCE_MINOR_VERSION
 echo -e "Source release version is: ${SOURCE_VERSION}\nSource minor version is: ${SOURCE_MINOR_VERSION}"
 echo -e "The source release version is gotten from clusterversion resource, that can not stand for the current version of worker nodes!"
-
+# Set genenral upgrade ci failure to overall as default
+export UPGRADE_FAILURE_TYPE="overall"
 if [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]; then
     run_command "oc get node -owide"
     rhel_repo
     rhel_pre_upgrade
     rhel_upgrade
-    rhel_post_upgrade
+    if ! rhel_post_upgrade; then
```

Comment:
Can you help update the line 262 to append `|| return 1` to ensure the failure can be caught.

Reply:
Similarly, for `update_cloud_credentials_oidc`, once the function is used in a `if` condition, any important lines in the function failed, need a explicit `return`, so that it can be caught by the `if` condition. So I guess we also need to append some `|| return 1` inside `update_cloud_credentials_oidc`

Reply:
add || return 1 in both update_cloud_credentials_oidc and rhel_post_upgrade to help catch important failure.